requirements.txt

```diff
@@ -1,5 +1,5 @@
-Fabric
 ansible
 pycrypto==2.5
 hvac
 click
```

Comment:
Is there a specific version of `click` or any of these other requirements that we need? Afraid of breaking changes as `ansible` or `click` gets updated.